app/app.iml

```diff
@@ -91,11 +91,11 @@
     <orderEntry type="library" exported="" name="support-v4-21.0.3" level="project" />
     <orderEntry type="library" exported="" name="play-services-base-6.5.87" level="project" />
     <orderEntry type="library" exported="" scope="TEST" name="hamcrest-core-1.3" level="project" />
-    <orderEntry type="library" exported="" scope="TEST" name="jmockit-1.15" level="project" />
     <orderEntry type="library" exported="" scope="TEST" name="junit-4.12" level="project" />
     <orderEntry type="library" exported="" name="butterknife-6.1.0" level="project" />
     <orderEntry type="library" exported="" name="appcompat-v7-21.0.3" level="project" />
     <orderEntry type="library" exported="" name="facebook-android-sdk-3.23.0" level="project" />
+    <orderEntry type="library" exported="" scope="TEST" name="jmockit-1.15" level="project" />
```

Comment:
This is problematic, the jmockit library should be before junit.
Seems like this is a bug in Android Studio: https://code.google.com/p/android/issues/detail?id=156708